keyboards/merge/um70/keymaps/via/rules.mk

```diff
@@ -1,3 +1,5 @@
 VIA_ENABLE = yes
 WPM_ENABLE = yes
-LTO_ENABLE = yes
\ No newline at end of file
+LTO_ENABLE = yes
+
+NKRO_ENABLE = no
```

Comment:
I'm not a fan of this.  NKRO is important to a lot of people.

Reply:
I have tested after enabling NKRO, and removing `#define RGBLIGHT_EFFECT_SNAKE`, it compiles under at 184 bytes free.

```Linking: .build/merge_um70_via.elf                                                                  [OK]
Creating load file for flashing: .build/merge_um70_via.hex                                          [OK]
Copying merge_um70_via.hex to qmk_firmware folder                                                   [OK]
Checking file size of merge_um70_via.hex                                                            [WARNINGS]

 * The firmware size is approaching the maximum - 28488/28672 (99%, 184 bytes free)```